playbooks/roles/scalardl/tasks/configuration.yml

```diff
@@ -12,3 +12,13 @@
     kubectl create secret docker-registry {{ kubernetes_registry_secret_name }} --docker-server={{ docker_registry }}
     --docker-username={{ docker_username }} --docker-password={{ docker_password }}
   when: not docker_registry_secrets.resources | length > 0
+
+- name: Install Helm Diff
+  kubernetes.core.helm_plugin:
+    state: present
+    plugin_path: https://github.com/databus23/helm-diff
+
+- name: Add scalardl charts repo
+  kubernetes.core.helm_repository:
+    name: scalardl
```

Comment:
Is there any problem with the repository name `scalardl`? If other than scalar-dl is also published in this chart repo, then `scalar-labs` might be appropriate.

Reply:
I just used `scalardl` as repo name because it used in the `scalardl` role.
Should we use `scalar-labs` ? I think using a different repo name for each ansible role will eliminate the influence between roles.

Reply:
Fixed the repo name as an option in 34f25c5.

Reply:
I feel scalar-labs is better since we are going to have a chart for scalardb as well soon.